src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -3272,6 +3280,7 @@ internal static bool IsSynthesizedAttributeOptional(WellKnownMember attributeMem
                 case WellKnownMember.System_STAThreadAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_AsyncStateMachineAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_IteratorStateMachineAttribute__ctor:
+                case WellKnownMember.System_Runtime_CompilerServices_IsByRefLikeAttribute__ctor:
```

Comment:
will try removing

---
In reply to: [117077872](https://github.com/dotnet/roslyn/pull/19411#discussion_r117077872) [](ancestors = 117077872)